app/screens/Search.js

```diff
@@ -90,7 +90,7 @@ function Search(props){
 
     return(
         <SafeAreaProvider style={styles.container}>
-            <NavBarComponent search={true} navigation={props.navigation}/>
+            <NavBarComponent search="true" navigation={props.navigation}/>
```

Comment:
Does this not need the { } around the prop?

Reply:
You don't need for strings or ints, I believe. I changed it on accident, but this still works.